packages/kie-sandbox-extended-services-image/Containerfile

```
@@ -20,18 +20,15 @@ ARG BUILDER_IMAGE_ARG
 
 FROM --platform=linux/amd64 ${BUILDER_IMAGE_ARG}
 
-ARG EXTENDED_SERVICES_DEFAULT_HOST
-ARG EXTENDED_SERVICES_DEFAULT_PORT
-
-ENV EXTENDED_SERVICES_HOST=$EXTENDED_SERVICES_DEFAULT_HOST
-ENV EXTENDED_SERVICES_PORT=$EXTENDED_SERVICES_DEFAULT_PORT
+ARG EXTENDED_SERVICES_DEFAULT_HOST = 0.0.0.0
+ARG EXTENDED_SERVICES_DEFAULT_PORT = 21345
```

```suggestion
ARG EXTENDED_SERVICES_DEFAULT_HOST=0.0.0.0
ARG EXTENDED_SERVICES_DEFAULT_PORT=21345
```

I don't think there should be spaces here